telethon/_client/messages.py

```diff
@@ -764,9 +764,11 @@ async def send_reaction(
     )
     result = await self(request)
     for update in result.updates:
-        if isinstance(update, _tl.UpdateMessageReactions):
-            return update.reactions
-    return reaction
+        if (
+            isinstance(update, _tl.UpdateMessageReactions) or
+            isinstance(update, _tl.UpdateEditMessage)
+        ):
+            return update.reactions  
```

Comment:
Shouldn't it be `update.message.reactions` for the edit?

Reply:
sorry,

Reply:
in channels, 

i think, we get both UpdateEditChannelMessage and UpdateMessageReactions

so, is the current code fine?

Reply:
I'm not sure, I haven't tested. If you want to try before merging let me know.

Reply:
currently, i am not able to test because getting some errors on the start() 😅